pkg/cmd/context/generate/build-configs.go

```diff
@@ -84,7 +87,7 @@ func BuildConfiguration(svcConfig *servicecontext.ServiceConfig, opts *options)
 		return opts.localizer.MustLocalizeError("context.generate.log.info.noSevices")
 	}
 
-	serviceAccount, err := createServiceAccount(opts, "to-do-generate-description")
+	serviceAccount, err := createServiceAccount(opts, fmt.Sprintf("rhoascli-%v", time.Now().Unix()))
```

Comment:
Wondering - should that be profile name instead? How we would be able to find that SA if not by it's name.

Reply:
It is important to be able to revoke it once it was exposed etc.

Reply:
That makes sense. Have updated the PR.